project/src/Game/World/World.gd

```diff
@@ -22,7 +22,7 @@ func _on_level_won() -> void:
 
 
 func next_level() -> void:
-	current_level += 1
+	current_level = Settings.level+1
```

Comment:
Zróbmy na Save'y i Loady osobny skrypt, jak będziemy chcieli zapisać coś więcej to będzie łatwiej to zmodyfikować.